Release 20210225 (#873)
publiccodenet/about: GOVERNANCE.md

# Content governance

This project represents the Foundation for Public Code.
Therefore any changes adopted into the `main` branch have to be accepted by the directors of the Foundation for Public Code.

> The `sprint` branch contains all features approved by at least one director and it represents the current 'working draft' version of the Foundation.

Reviewer: ```suggestion
The `develop` branch contains all features approved by at least one other committer and it represents the current 'working draft' version of the Foundation.
```

> ## New content and changes into the `sprint` branch

Reviewer: ```suggestion
## New content and changes into the `develop` branch
```

Every change and new feature can be accepted into the `develop` branch by any director that did not open that pull request.

> ## Publishing changes from `sprint` to `main`

Reviewer: ```suggestion
## Publishing changes from `develop` to `main`
```
